### PR TITLE
[OYPD-534] Add GroupEx Pro paragraphs behat test.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-534
D.o https://www.drupal.org/node/2893904

Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Steps for review

- [x] Review test code
- [x] Validate test passes

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1848 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1848 |
| Installation process | http://install.openy-dev.ffwua.com/build1848/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1848/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1848

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1848/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1848/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1848/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1848/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1848/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1848/profiles/contrib/openy/build/reports/behat

**Comment by AlexNetman** [comment]
Let's add tests also for data that are displayed. Data in sandbox are same for each Monday years ahead. And same for each Tue, Wed ... 
So we can rely on them.

1) Check if needed data is displayed by default. 
Change date to Saturday or Sunday and check how data changed.
Change location and check updated data.
2) Try to click on trainer name and check results
3) Try to click on Class name or Select in dropdown and see results. Also class description should appear.
4) Test that "Download PDF" link is available.

**Comment by alexschedrov** [comment]
@AlexNetman @Kerby70 Let's do this if it's possible and won't take much time.

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1924 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1924 |
| Installation process | http://install.openy-dev.ffwua.com/build1924/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1924/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1924

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1924/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1924/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1924/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1924/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1924/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1924/profiles/contrib/openy/build/reports/behat

**Comment by alexschedrov** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1927 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1927 |
| Installation process | http://install.openy-dev.ffwua.com/build1927/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1927/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build1927

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1927/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1927/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1927/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1927/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1927/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1927/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

File: tests/features/paragraphs/groupex_pro_paragraphs.feature [hunk 1]
```diff
@@ -31,6 +31,54 @@ Feature: GroupEx Pro paragraphs
...
+    And I should see text matching "8:30am 60 min"
+    And I should see text matching "Chisel Studio 1"
+    And I should see text matching "Jennifer K."
+    And I should see text matching "Chisel Visit Strength with Jennifer K. Class will take place at Studio 1. GXP Club - Fable Jennifer K."
+    And I should see text matching "11:00am 60 min"
+    And I should see text matching "Yoga GX2"
+    And I should see text matching "John F."
+    And I should see text matching "Yoga Visit Yoga with John F. Class will take place at GX2. GXP Club - Fable John F."
+    And I should see text matching "7:00pm 60 min"
+    And I should see text matching "3D XTREME­&trade; GX1"
```
Comment (AlexNetman) [inline tests/features/paragraphs/groupex_pro_paragraphs.feature:48]: This " &trade" should be rendered as ™  . 
So I guess we should fix the error. And update test on all those places.

**Pull request merged**